lib/ansible/modules/cloud/amazon/iam_role_facts.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
```

Comment:
Is `certified` correct?

Reply:
Umm, absolutely not. I wonder what I copied *that* from.

Reply:
I hope Ansible know that `efs_facts` is a certified support module ;)